misc/python/materialize/query_fitness/all_parts_essential.py

```diff
@@ -23,7 +23,7 @@
 
 from typing import Optional
 
-from pg8000.native import DatabaseError  # type: ignore
+from pg8000.dbapi import DatabaseError
```

Comment:
question for my own education - when should I use `pg8000.dbapi` vs when when should i import from `pg8000` directly? perhaps the right answer is "read this book about how to write python" which im also down to try

Reply:
You should import from wherever the documentation says to! In this case it's a bit confusing, because the type exists at both `pg8000.native` and `pg8000.dbapi`, but we're using the `dbapi` connector, not the `native` connector, so it makes a bit more sense to import from the latter.